examples/clicReconstruction.xml

```diff
@@ -447,14 +448,26 @@
     <!--parameter name="Verbosity" type="string">DEBUG </parameter-->
   </processor>
 
+  <processor name="ClonesAndSplitTracksFinder" type="ClonesAndSplitTracksFinder">
+    <parameter name="InputTrackCollection" type="string"> SiTracks </parameter>
+    <parameter name="OutputTrackCollection" type="string"> SiTracksMerged </parameter>
+    <parameter name="MultipleScatteringOn" type="bool"> true </parameter>
+    <parameter name="EnergyLossOn" type="bool"> true </parameter>
+    <parameter name="SmoothOn" type="bool"> false </parameter>
+    <parameter name="extrapolateForward" type="bool"> true </parameter>
+    <parameter name="maxDeltaTheta" type="double"> 0.59 </parameter>
+    <parameter name="maxDeltaPhi" type="double"> 0.99 </parameter>
+    <parameter name="maxDeltaPt" type="double"> 0.69 </parameter>
+  </processor>
+
   <processor name="Refit" type="RefitFinal">
     <!--Refit processor that calls finaliseLCIOTrack after taking the trackstate from the existing track. No re-sorting of hits is done-->
     <!--Use Energy Loss in Fit-->
     <parameter name="EnergyLossOn" type="bool"> true </parameter>
     <!--Name of the input track to MCParticle relation collection-->
     <parameter name="InputRelationCollectionName" type="string" lcioInType="LCRelation"> SiTrackRelations </parameter>
     <!--Name of the input track collection-->
-    <parameter name="InputTrackCollectionName" type="string" lcioInType="Track"> SiTracks </parameter>
+    <parameter name="InputTrackCollectionName" type="string" lcioInType="Track"> SiTracksMerged </parameter>
```

Comment:
This make it incompatible with the TruthTracing output, please change the output from ConformalTracking to SiTracksCT and the output from ClonesAndSplitTracks to SiTracks